scripts/functions/gemset

```diff
@@ -174,7 +176,24 @@ gem_install_force()
   then install_params+=( -v "${gem_version}" )
   fi
 
-  install_params+=( --no-ri --no-rdoc $rvm_gem_options )
+  case "${rvm_ruby_string}" in
+    ruby-head*)
+      install_params+=( --no-document )
+      ;;
+
+    ruby-*)
+      if __rvm_version_compare "$rvm_ruby_version" -ge 2.6
+      then install_params+=( --no-document )
+      else install_params+=( --no-ri --no-rdoc )
+      fi
+      ;;
+
+    *)
+      install_params+=( --no-ri --no-rdoc )
```

Comment:
This will become an issue for other engines as well when they adopt RubyGems 3.

Reply:
Yeah, I was thinking about it...

Do you think we should simply do?

```
      if __rvm_version_compare "$(\command \gem --version)" -ge 2.2
      then install_params+=( --no-document )
      else install_params+=( --no-ri --no-rdoc )
      fi
```

Reply:
Done. Let's see what reporter will say...